examples/csv-streams-to-record-arrays/csv_streams_to_record_array.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/data.csv;
+import ballerina/io;
+
+type Book record {
+    string name;
+    string author;
+    decimal price;
+    string publishedDate;
+};
+
+type BookMinimal record {|
+    string name;
+    string author;
+|};
+
+const csvFilePath = "./files/csv_file.csv";
+
+public function main() returns io:Error? {
+    // Write a CSV file.
+    check io:fileWriteCsv(csvFilePath, [["name", "author", "price", "publishedDate"],
+                                        ["Effective Java", "Joshua Bloch", "45.99", "2018-01-01"],
+                                        ["Clean Code", "Robert C. Martin", "37.5", "2008-08-01"]]);
+
+    // Read the CSV file as a `byte stream`.
```

Comment:
Why the backticks with byte stream? We use backticks only with keywords, names, etc. in the code.